service/controller/clusterapi/v29/drainer_resource_set.go

```diff
@@ -106,25 +104,10 @@ func NewDrainerResourceSet(config DrainerResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	var tccpOutputsResource controller.Resource
```

Comment:
We do not need the outputs resource anymore here since the only relevant information is the ASG name and we do this differently now.